.storybook/preview.js

```diff
@@ -1,9 +1,10 @@
+import "../src/App.css";
+import "../src/Colors.css";
+
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
-  controls: {
-    matchers: {
-      color: /(background|color)$/i,
-      date: /Date$/,
-    },
+  backgrounds: {
+    default: "myApp",
+    values: [{ name: "myApp", value: "#2b0080" }],
```

Comment:
Nice.

Reply:
I really like it when you hint something and I have to look it up and it works at the end!!